srcpkgs/slurp/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'slurp'
+pkgname=slurp
+version=1.0
+revision=1
+build_style=meson
+hostmakedepends="pkg-config"
```

Comment:
I recommend adding `scdoc` to hostmakedepends=`, meson will create manpages (very useful!) for the project.

https://github.com/emersion/slurp/blob/master/meson.build#L48